### ISSUE TITLE
Amend 'allowedChild' prop for Dijit containers

### ISSUE THREAD

**Issue by jhpedemonte** [maqetta/maqetta#755]
Commit ecdd127f4f5b7b54e980 changed the `allowedChild` property for **BorderContainer**, **AccordionContainer** and **TabContainer** to be just `dijit.layout.ContentPane`.  This was done mainly to remove `"allowedChild": "ANY"`, which implies HTML children and is incorrect for these container widgets.

However, they allow a wider selection of widgets than just `ContentPane`, so we need to amend the `allowedChild` prop for these containers.  Not sure what the rule is -- it may just be any Dijit widget.  Seems that the only requirement (according to the code) is for the widget to have a `startup()` property -- and that is any Dijit widget.

Easiest solution is to define a `class` for the affected widgets.  So maybe add `DijitWidget` to all such widgets, and then change allowedChild to `"allowedChild": "DijitWidget"` for the 3 containers mentioned above.

@peller @JonFerraiolo

**Comment by peller** [comment]
yes, any dijit widget should be allowed, though Maqetta content will always have the extra content pane, and that's probably ok.  startup() is a no-op unless the widget needs to do some processing after the parents are established in the DOM.

**Comment by JonFerraiolo** [comment]
@doronrosenberg
